rust/main/chains/hyperlane-ton/src/traits/ton_api_center.rs

```diff
@@ -57,6 +56,11 @@ pub trait TonApiCenter {
         address: String,
         include_boc: bool,
     ) -> ChainResult<AccountStateResponse>;
+    async fn get_wallet_information(
+        &self,
+        address: &str,
+        include_boc: bool,
```

Comment:
The name of this arg is different from 'use_v2' ( see impl)

Reply:
fixed